spec/classroom_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rspec'
+require_relative '../classroom'
+require_relative '../student'
+
+describe 'Classroom' do
+  before :all do
+    @classroom = Classroom.new('Ruby')
+    @student1 = Student.new(@classroom, 18, 'Alice', true)
+    @student2 = Student.new(@classroom, 19, 'Bob', true)
+  end
+
+  it 'has a label' do
+    expect(@classroom.label).to eq 'Ruby'
+  end
```

Comment:
- [ ] Just like the previous reviewer pointed out, you have failing tests - about 14 of them. Please try to fix them and make them green ✔️. I have attached a snapshot of the failing tests below 👇 
![image](https://user-images.githubusercontent.com/46839548/233615192-a72283f1-d9aa-415b-a301-2209e137cea6.png)